tigergraph/ddl/load_dynamic.gsql

```diff
@@ -23,55 +23,159 @@ CREATE LOADING JOB load_dynamic FOR GRAPH ldbc_snb {
   DEFINE FILENAME file_Post_hasCreator_Person;
   DEFINE FILENAME file_Post_hasTag_Tag;
   DEFINE FILENAME file_Post_isLocatedIn_Country;
+  //load schema group 1
+  // load dynamic vertex
+  LOAD file_Comment 
+    TO VERTEX Comment1 VALUES ($1, ToMiliSeconds($0), $2, $3, $4, $5) USING header="true", separator="|";
+  LOAD file_Post
+    TO VERTEX Post1 VALUES ($1, $2, ToMiliSeconds($0), $3, $4, $5, $6, $7) USING header="true", separator="|";
+  LOAD file_Forum
+    TO VERTEX Forum1 VALUES ($1, $2, ToMiliSeconds($0), _) USING header="true", separator="|";
+  LOAD file_Person
+    TO VERTEX Person1 VALUES ($1, $2, $3, $4, ToMiliSeconds($5), ToMiliSeconds($0), $6, $7, SPLIT($8,";"), SPLIT($9,";"), _) USING header="true", separator="|";
+
+  // load dynamic edge
+  LOAD file_Forum_containerOf_Post
+    TO EDGE CONTAINER_OF1 VALUES ($1, $2) USING header="true", separator="|";
```

Comment:
LOAD file_Forum_containerOf_Post
    TO EDGE CONTAINER_OF VALUES ($1 Forum1, $2 Post1) USING header="true", separator="|"
    TO EDGE CONTAINER_OF VALUES ($1 Forum2, $2 Post2) USING header="true", separator="|"
    TO EDGE CONTAINER_OF VALUES ($1 Forum3, $2 Post3) USING header="true", separator="|";